### PR TITLE
Adding Multiple Programming Language Courses

### PR DESCRIPTION
**Pull request by GauriBhandari**
### 🛠️ Fixes Issue
Issue- #1468 

### 👨‍💻 Changes proposed

I have added courses for the programming languages, catering to diverse learners and industry demands.
Added courses for the following programming languages:
1. Java
2. React
3. Python
4. Rust
5. Shell
6. SQL
7. Typescript
8. PHP
9. Scala
10. Perl
11. Lisp

If there are any specific programming languages or technology domains that you believe would be beneficial to include, I'd be more than happy to work on expanding the course catalog. If there are any guidelines or preferences regarding course additions, please let me know. 

Thank you for your time and consideration. I'm looking forward to your feedback.

### ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by rohanbabbar04** [comment]
- Look into [this](https://deploy-preview-1474--aec-library-wbsite.netlify.app/resource), there are some broken image links...
- Also I don't think you additions match the pattern we are trying to implement. I would want the user to move right-left to look for his preferred course I think that looks better, currently it looks crowded.
- Also make sure the size of images are same...

**Review by rohanbabbar04 — CHANGES_REQUESTED** [review]

File: resource.html [hunk 1]
```diff
@@ -217,14 +232,22 @@ <h3 class="hover-underline-animation ">Programming Languages Courses <img class=
...
+
                             src="https://hackernoon.com/images/VyvcKdbWHbTaN3QzRCQQS7pXASq1-9x7p315b.jpeg" alt="cpp">
+
                     </a>
 
                     <a href="https://www.udemy.com/course/learn-flutter-dart-to-build-ios-android-apps/"
                         target="_blank">
                         <img width="33%" height="300"
 
+
```
[the remark this answers is not in the document]
Comment (GauriBhandari) [reply resource.html]: The extra lines were removed

**Comment by GauriBhandari** [comment]
The issue regarding the broken images and the course layout has been resolved. If there are any further improvements needed please let me know. 
Thank you!

**Comment by GauriBhandari** [comment]
hi @rohanbabbar04 ,
I hope you're doing well. I just wanted to check in on the status of my Pull Request. It seems that it hasn't been reviewed yet, and I'm eager to receive your feedback on the changes I made.

File: resource.html [hunk 2]
```diff
@@ -1325,13 +1291,22 @@ <h2 class="hover-underline-animation">Regional Language Book Section</h2>
...
                 <div class="social-container">
                     <div class="social-icons">
+
+                            <a class="Twitter" href="https://twitter.com/mesourav44" target="_blank"><i class="fab fa-twitter"></i></a>
+                            <a class="Instagram" href="https://www.instagram.com/mesouravofficial/" target="_blank"><i class="fab fa-instagram"></i></a>
+                            <a class="linkedin" href="https://www.linkedin.com/in/sauravmukherjee44/" target="_blank"><i class="fab fa-linkedin"></i></a>
+                            <a class="Github" href="https://github.com/SauravMukherjee44/" target="_blank"><i class="fab fa-github-square"></i></a>
+                            <a class="Blog" href="#" target="_blank"><i class="fa fa-rss-square"></i></a>
+                    </div>          
+
```
Comment (rohanbabbar04) [inline resource.html:1301]: Why has this been added ?

Comment (GauriBhandari) [reply resource.html:1301]: I apologize for the misunderstanding. This code snippet was not added by me, it was already their. If you find the code snippet unnecessary for the project, please let me know,  I'll remove it.

Comment (rohanbabbar04) [reply resource.html:1301]: I thinks its been added by you(as it shows over here) maybe when you were merging main you handled it like this 
Bcoz the code of this is already below...

Comment (rohanbabbar04) [reply resource.html:1301]: So needs to be removed

Comment (GauriBhandari) [reply resource.html]: The code snippet has been successfully removed.
I kindly request you to review the updated Pull Request and consider merging it into the main branch if everything looks good.

**Comment by GauriBhandari** [comment]
The latest commit now includes only the relevant changes without any formatting modifications.

**Comment by GauriBhandari** [comment]
hi @rohanbabbar04  ,
I just wanted to check in on the status of my Pull Request. It seems that it hasn't been reviewed yet, and I'm eager to receive your feedback on the changes I made.

**Comment by rohanbabbar04** [comment]
Looks good thanks

**Pull request merged**